crmsh/bootstrap.py

```diff
@@ -1927,7 +1927,8 @@ def setup_passwordless_with_other_nodes(init_node):
     # Fetch cluster nodes list
     remote_user = _context.user_list[0]
     local_user = _context.current_user
-    cmd = 'ssh {} {}@{} PATH=\\"\\$PATH\\":/usr/sbin:/sbin crm_node -l'.format(SSH_OPTION, remote_user, init_node)
+    remote_sudo = "" if remote_user == 'root' else "sudo "
+    cmd = f'ssh {SSH_OPTION} {remote_user}@{init_node} PATH=\\"\\$PATH\\":/usr/sbin:/sbin {remote_sudo}crm_node -l'
```

Comment:
We can just use `sudo crm_node -l` here. It is not needed any longer to append sbin directories to $PATH. And I think it is OK to prepend sudo even if remote_user is root.

Reply:
Changed